### DISCUSSION TITLE
[Feature Request] Config/Override default transition

### DISCUSSION THREAD

**Discussion by conduongtong** [heroui-inc/heroui#4735] [Feature requests]
### Is your feature request related to a problem? Please describe.

Currently cannot override default transition because it fixed in [nextui-org/nextui/packages/core/theme/src/utilities/transition.ts](https://github.com/nextui-org/nextui/blob/canary/packages/core/theme/src/utilities/transition.ts)

### Describe the solution you'd like

Can change default transition in tailwind config

### Describe alternatives you've considered

Can change default transition in tailwind config

**Comment by linear** [comment]
<p><a href="https://linear.app/nextui-inc/issue/ENG-1311/[feature-request]-configoverride-default-transition">ENG-1311 [Feature Request] Config/Override default transition</a></p>
